tests/robot-tests/tests/libs/slack.py

```diff
@@ -0,0 +1,86 @@
+import os
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+import shutil
+import json
+import requests
+from bs4 import BeautifulSoup
+
+PATH = f'{os.getcwd()}{os.sep}test-results'
+
+
+def _generate_slack_attachments():
+    failed_tests = dict()
+    passed_tests = dict()
+
+    report = open(f'{PATH}{os.sep}output.xml', 'rb')
+    contents = report.read()
+
+    soup = BeautifulSoup(contents, 'xml')
+    test = soup.find('total').find('stat')
+
+    failed_tests = test['fail']
+    passed_tests = test['pass']
+
+    return [
+        {
+            "pretext": "All results",
+            "color": "danger" if failed_tests else "good",
+            "mrkdwn_in": [
+                "pretext"
+            ],
+            "fields": [
+                {
+                    "title": "Total test cases",
+                    "value": int(passed_tests) + int(failed_tests)
```

Comment:
I'd do these int conversions where `failed_tests` and `passed_tests` are instantiated. So,
```
failed_tests = int(test['fail'])
passed_tests = int(test['pass'])
```